server/games.js

```diff
@@ -1,8 +1,31 @@
+var db = require('../database/index.js');
 var express = require('express');
 var app = express();
 
-app.get('/', function (req, res) {
-  res.send('Hello World!')
+app.get('/game/:id', function (req, res) {
```

Comment:
I notice you're using arrow functions a lot through the code. I think it might make sense to use the arrow function here instead of the `function (req, res)` pattern to just be consistent. This is just a style comment, so up to you if you want to change it